addons/stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -241,14 +245,17 @@ def action_done(self):
                     picking.batch_id.id,
                     picking.batch_id.name))
 
+        # Run sanity_check as a batch and ignore the one in button_validate() since it is done here.
+        pickings._sanity_check(separate_pickings=False)
```

Comment:
if picking must be checked separately in this case I would prefere doing 
```python
for pick in pickings:
    pick._sanity_check()
```
to be able to remove the extra params. Do you think it would the same ?